integration_test.go

```diff
@@ -91,6 +91,10 @@ func setup(t *testing.T, ctx context.Context, dmls []string) (*Session, string,
 		t.Fatalf("failed to create table: err=%s", err)
 	}
 
+	// Wait until created table becomes visible.
+	// TODO(furuyama): Change to a sophisticated way for checking table creation
+	time.Sleep(10 * time.Second)
```

Comment:
Just curious: is there a chance that when `op.Wait(ctx)` is finished, the created table is still not visible?

Reply:
I didn't dig into the details, but it looked so.

Without time.Sleep(10 sec), I got a following error occasionally at touching the database.
`spanner: code = "Aborted", desc = "Transaction aborted. Database schema probably changed during transaction, retry may succeed."`

Reply:
Ok. I see. If that is the case, adding a retrying here probably would be a bit nice. But you can do it later.

Reply:
Sure, I'll change that to do retry in the next pull request. Thank you!